.github/workflows/reusable-run-go-acceptance-test.yml

```diff
@@ -0,0 +1,42 @@
+name: Run Reusable Go Acceptance Test
+
+on:
+  workflow_call:
+    inputs:
+      name:
+        description: 'The name to use that will appear in the output log file artifact'
+        required: true
+        type: string
+      path:
+        description: 'The path to the test without the precedeing "./" or following "/..." e.g. go test -v ./$path/...'
+        required: true
+        type: string
+    # We will need to add the capacity for receiving passed secrets once we get to the tests that require API credentials
+
+env:
+  VAULT_ACC: 1
```

Comment:
I wonder if the caller workflow should set this? That way we could have a single workflow that runs a go test that we call into, but whether or not ACC is enabled is up to the caller.